lib/rules/color-hex-case/index.js

```diff
@@ -50,6 +61,7 @@ const rule = function (expectation) {
           index: match.startIndex,
           result,
           ruleName,
+          suggestedFix: config && config.suggestFixes && suggestFix(match.startIndex, hexValue, expectedHex),
```

Comment:
This can be cleaner when we redefine Flow types for the new third argument. We can do that later.